backend/dataall/modules/metadata_forms/db/metadata_form_models.py

```diff
@@ -0,0 +1,149 @@
+from sqlalchemy import (
+    Column,
+    String,
+    Integer,
+    ForeignKey,
+    Boolean,
+    ForeignKeyConstraint,
+    PrimaryKeyConstraint,
+)
+from sqlalchemy.dialects.postgresql import ARRAY
+from sqlalchemy.orm import relationship, validates
+
+from dataall.base.db import Base, utils
+
+from dataall.modules.metadata_forms.db.enums import MetadataFormFieldType
+
+
+class MetadataForm(Base):
+    __tablename__ = 'metadata_form'
+    uri = Column(String, primary_key=True, default=utils.uuid('metadata_form'))
+    name = Column(String, nullable=False)
+    description = Column(String, nullable=False)
+    SamlGroupName = Column(String, nullable=True)
```

Comment:
should this be nullable?

Reply:
Nope. Good catch

Reply:
[5539e10](https://github.com/data-dot-all/dataall/pull/1422/commits/5539e10bd0d56f6b382f0aa476d24fec0644f14c)